frame/collator-selection/src/tests.rs

```diff
@@ -117,7 +117,9 @@ fn cannot_register_candidate_if_too_many() {
 
         // reset desired candidates:
         <crate::DesiredCandidates<Test>>::put(1);
-        assert_ok!(CollatorSelection::register_as_candidate(RuntimeOrigin::signed(4)));
+        assert_ok!(CollatorSelection::register_as_candidate(
```

Comment:
Why are unrelated files changed by the formatter? Are you using different settings?

Reply:
I guess it’s new astar-frame formatting settings